interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: this is a protected word inside of javascript used to access key values inside of objects
 
-  Researched answer:
+  Researched answer:This points to an object and allows you to access key values and methods stated within the object.
 
 
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: Yarn is a collection of packages that help your code import necessary files to execute certain actions such as testing or existing outside of your local directory. In react yarn modifies all the node files when you run yarn.
 
-  Researched answer:
+  Researched answer:Yarn is a package manager for code, Yarn inside of React addsa directory of node-modules and also generates the yarn-.lock file which contains info about node module dependencies and thier versions.
 
 
 
 3. What are props in React?
 
-  Your answer:
+  Your answer: Props are a way to pass properties of a parent class to child classes inside of a react app
 
-  Researched answer:
+  Researched answer: Think of props as being similar to a global variable or object that react allows us to pass information between components with.
 
 
 
 4. What is JSX?
 
-  Your answer:
+  Your answer: JSX stands for JavaScript XML which is a mixture of the javascript and html language that exist within react
 
-  Researched answer:
+  Researched answer: JSX is a syntax extension to JavaScript that is used in React.
 
 
 
 5. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event is a change in data within a react app
 
-  Researched answer:
+  Researched answer: A DOM event is a signal that something has occured or is ocurring.
```

Comment:
You may want to look into DOM events in HTML and how they correspond to React's virtual DOM.